sdk/tables/aztable/recordings/TestTableClient_Cosmos/TestAddAnnotatedEntity-variables.yaml

```diff
@@ -1,5 +1,5 @@
 ---
 COSMOS_TABLES_ENDPOINT_SUFFIX: cosmos.azure.com
-TABLES_COSMOS_ACCOUNT_NAME: chrisstablesprim
+TABLES_COSMOS_ACCOUNT_NAME: seankaneprim
 TABLES_PRIMARY_COSMOS_ACCOUNT_KEY: Kg==
-randomSeed: "1621634379"
```

Comment:
Why does this end up changing, btw? Is it auto-generated as well?

Reply:
I had to re-record these tests

Reply:
The random seed used was different than when Chris originally ran it, I re-record by completely wiping the recordings files and then re-running tests.

Reply:
Just for additional context (in case you didn't already know this), the randomSeed is a recorded value that is used by tests in playback mode so that any randomly generated recorded values such as names or UUIDs are generated consistently.